modules/helm-understanding-helm.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id='understanding-helm_{context}']
+= Understanding Helm
+
+Helm is a CLI tool the simplifies deployment of applications and services to {prod-title} clusters. Helm uses a packaging format called charts - a collection of files that describe {prod-title} resources. 
+
+== Key features
+
+* Searching through a large collection of charts at chart repositories.
+* Modifying existing charts.
+* Creating your own charts with {prod-title} or Kubernetes resources.
+* Packaging and sharing your applications as charts.
```

Comment:
Key Features
Helm provides the ability to:

- search through a large collection of charts stored in the chart repository.
- modify existing charts.
- create...
- package and share...

Reply:
Thanks, reworded.